src/main/java/lv/ctco/javaschool/eventorganaizer/boundary/EventOrganizationApi.java

```diff
@@ -183,4 +181,25 @@ public void deleteEvent(@PathParam("id") Long id) throws IllegalArgumentExceptio
     public void deletePoll(@PathParam("id") Long id) throws IllegalArgumentException {
         pollStore.deletePollById(id);
     }
+
+
+    @POST
+    @RolesAllowed({"ADMIN", "USER"})
+    @Path("/{id}/saveFeedback")
+    public void saveFeedback(String feedbackText, @PathParam("id") Long id) {
+        Feedback feedback = new Feedback();
+        feedback.setEvent(eventStore.getEventById(id).get());
+        feedback.setFeedbackAuthor(userStore.getCurrentUser().getUsername());
+        feedbackText=feedbackText.replace('"',' ');
```

Comment:
What is this needed for?

Reply:
When I display feedback output string displays with "". With this:   feedbackText.replace('"',' ')    i remove "".

Reply:
This is a potential bug, and normally it would need to be fixed. For now I'll approve since the time is short.

Reply:
Thank you.